src/components/Common/Module/FieldPicker.vue

```diff
@@ -1,49 +1,143 @@
 <template>
-    <fieldset class="form-group">
-      <div class="fields d-flex">
-        <div class="available">
-          <label>{{ $t('selector.available') }}</label>
-          <b-button @click.prevent="selectedFields = [...allFields]" variant="link" class="float-right">{{ $t('selector.selectAll') }}</b-button>
-          <draggable
-            class="drag-area border"
-            :list.sync="availableFields"
-            :group="group"
+    <fieldset
+      :disabled="disabled"
+      class="form-group"
+    >
+      <b-form-group
+        :label="$t('block:recordList.fields')"
+        class="mt-3 h5"
+        label-cols="2"
```

Comment:
Didn't we say to remove this ? Designs show search directly above the picker not inline with label